packages/aws-cdk-lib/aws-codepipeline-actions/test/stepfunctions/stepfunctions-invoke-actions.test.ts

```diff
@@ -2,153 +2,160 @@ import { Template, Match } from '../../../assertions';
 import * as codepipeline from '../../../aws-codepipeline';
 import * as s3 from '../../../aws-s3';
 import * as stepfunction from '../../../aws-stepfunctions';
-import { Stack } from '../../../core';
+import { Stack, Stage } from '../../../core';
 import * as cpactions from '../../lib';
 
 describe('StepFunctions Invoke Action', () => {
-  describe('StepFunctions Invoke Action', () => {
-    test('Verify stepfunction configuration properties are set to specific values', () => {
-      const stack = new Stack();
-
-      // when
-      minimalPipeline(stack);
-
-      // then
-      Template.fromStack(stack).hasResourceProperties('AWS::CodePipeline::Pipeline', Match.objectLike({
-        Stages: [
-          //  Must have a source stage
-          {
-            Actions: [
-              {
-                ActionTypeId: {
-                  Category: 'Source',
-                  Owner: 'AWS',
-                  Provider: 'S3',
-                  Version: '1',
-                },
-                Configuration: {
-                  S3Bucket: {
-                    Ref: 'MyBucketF68F3FF0',
-                  },
-                  S3ObjectKey: 'some/path/to',
-                },
+  test('Verify stepfunction configuration properties are set to specific values', () => {
```

Comment:
The `describe` block was duplicated, so I removed the duplicate... This caused a bunch of indent changes down there, so I've commented on what's "really" changed here so reviewers can focus on that.